app/styles/app.scss

```diff
@@ -2,41 +2,88 @@ $blue: #4c66b2;
 $white: white;
 $red: #ff0033;
 
-body{
+
+//==========  Mobile First Method  ==========
+
+  // Custom, iPhone Retina
+@media only screen and (min-width : 320px) {
+  .btn-danger {
+    width: 100%;
+    float: none;
+  }
+
+}
+
+  // Extra Small Devices, Phones
+@media only screen and (min-width : 480px) {
+
+}
+
+  // Small Devices, Tablets
+@media only screen and (min-width : 768px) {
+  .navbar-nav {
+    float: right;
+  }
+
+}
+
+  // Medium Devices, Desktops
+@media only screen and (min-width : 992px) {
+  .btn-danger {
+    width:180px;
+    height: 100px;
+  }
+
+}
+
+  // Large Devices, Wide Screens
+@media only screen and (min-width : 1200px) {
+
+}
+
+body {
   color: $blue;
   background-size: cover;
+  width: 100%;
 }
 
-.colorfix{
-  background-color: $blue;
+.content {
+    background-color: $blue;
 }
 
-.header-links{
+.colorfix {
+    background-color: $blue;
 }
 
-.btn-head{
-  width: 180px;
-  height: 100px;
-  font-size: 30px;
+.btn-head {
+    font-size: 100%;
 }
 
-.pad-head{
-  padding: 30px 10px 0px 10px;
+.pad-head {
+    padding: 30px 10px 0px 10px;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Shorthand form for property `padding` should be written more concisely as `30px 10px 0px` instead of `30px 10px 0px 10px`<br>`0px` should be written without units as `0`